react/training-project/src/app/screens/PrimaryLayout/screens/Profile/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { CSSTransition } from 'react-transition-group';
+import PropTypes from 'prop-types';
+
+import defaultPhoto from '~assets/hacker.png';
+
+import { WON, LOST, DRAW } from './constants';
+import style from './styles.scss';
+
+function Profile({ name, photoUrl, charge, description, won, lost, draw }) {
+  return (
+    <CSSTransition
+      in
+      appear
+      timeout={300}
+      classNames={{
+        appear: style.profileAppear,
+        appearActive: style.profileAppearActive
+      }}
+    >
+      <div className={style.profileContainer}>
+        <CSSTransition
+          in
+          appear
+          timeout={900}
+          classNames={{
+            appear: style.photoAppear,
+            appearActive: style.photoAppearActive
+          }}
+        >
+          <img className={style.photo} src={!photoUrl ? defaultPhoto : photoUrl} alt="profile" />
```

Comment:
I would do `src={photoUrl || defaultPhoto}`.
Or just `src={photoUrl}` and set the defaultPhoto in the defaultProps.